onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -2946,6 +2946,24 @@ TEST_F(GraphTransformationTests, TransposeMatmulTransBatchNoFusion) {
   }
 }
 
+TEST_F(GraphTransformationTests, GH24341) {
```

Comment:
Change a test name like TransposeMatmulSameInputNoFusion to make consistent with other test cases? You can put comment for github issues.

Reply:
I was following this convention like previously did in https://github.com/microsoft/onnxruntime/pull/11116/files#diff-7ebe64fe3211b862da76388b3a7a3e281a40848bdafab576a9071bc9fbb24b15R35 or https://github.com/microsoft/onnxruntime/pull/11839/files#diff-2061b610e6e8d0211ebb4f304684feb30a1875de130044898377c948e4871512. Not sure if this is the preferred way.

I am OK with either and please let me know what do you think

Reply:
How about `TransposeMatmulSameInput_gh_issue_24341` like

https://github.com/microsoft/onnxruntime/blob/6a25fade269d965ff01775440ed6e4d0d25f83da/onnxruntime/test/optimizer/graph_transform_test.cc#L7740